x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/endpoint_package_custom_extension.test.tsx

```diff
@@ -21,6 +21,12 @@ describe('When displaying the EndpointPackageCustomExtension fleet UI extension'
   let render: () => ReturnType<AppContextTestRender['render']>;
   let renderResult: ReturnType<AppContextTestRender['render']>;
   let http: AppContextTestRender['coreStart']['http'];
+  const artifactCards = [
```

Comment:
can you make this immutable or use a function to return the array, so that state is not shared between tests?